README.rst

```diff
@@ -195,6 +195,43 @@ The ``test`` method uses the SciTokens built-in path parsing to validate the
 authorization.  The ``generate_acls`` method allows the caller to cache
 the ACL information from the token.
 
+Creating Sample Tokens
+----------------------
+
+Typically, an access token is generated during an OAuth2 workflow to facilitate 
+authentication and authorization. However, for testing and experimentation purposes, 
+`our token generator <https://demo.scitokens.org/>`__ provides users with the
+ability to create sample tokens with customized payload:
+
+::
+    
+    >>> payload = {"sub": "<email adress>", "scope": "read:/protected"}
+    >>> token = scitokens.utils.demo.token(payload)
+
+The ``token`` method makes a request to the generator to create a serialized token 
+for the specified payload. Users can also retrieve a parsed token by calling the 
+``parsed_token`` method, which returns a SciToken object corresponding to the 
+payload. The object contains the decoded token data, including the claims and signature. 
+
+Decorator
+-------------
+
+This protect decorator is designed to be used with a flask application. It can be used like:
```

Comment:
```suggestion
This protect decorator is designed to be used with a `flask <https://flask.palletsprojects.com/>`_ application. It can be used like:
```